content/docs/deployment/csmoperator/modules/replication.md

```diff
@@ -16,12 +16,12 @@ To use Replication, you need at least two clusters:
 
 To configure all the clusters, follow the steps below:
 
-1. On your main cluster, follow the instructions available in CSM Replication for [Installation using repctl](../../../replication/deployment/install-repctl.md). NOTE: On step 4 of the link above, you MUST use the command below to automatically package all clusters' `.kube` config as a secret:           
+1. On your main cluster, follow the instructions available in CSM Replication for [Installation using repctl](../../../../replication/deployment/install-repctl). NOTE: On step 4 of the link above, you MUST use the command below to automatically package all clusters' `.kube` config as a secret:           
```

Comment:
This should not be a note. This should be a distinct step instruction. Change to read "On your main cluster, follow the instructions available in CSM Replication for [Installation using repctl](../../../../replication/deployment/install-repctl), with the exception of step 4. When you reach step 4, you MUST use the command below to automatically package all clusters' `.kube` config as a secret:"

Reply:
We will address this as recommended. Thanks.